clang/lib/Sema/SemaExceptionSpec.cpp

```diff
@@ -574,6 +675,7 @@ static bool CheckEquivalentExceptionSpecImpl(
     }
   }
 
+#if 0
```

Comment:
Hmm?

Reply:
Just an implementation remnant, I'll remove it :)